docs/DeveloperGuide.md

```diff
@@ -283,28 +283,162 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `InterviewHub` and the **Actor** is the `hiring manager`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 Add an interview**
```

Comment:
Nice work labelling the use cases as taught in class. It makes for a really nicely formatted DG! 👍 👍